dialect/sql/builder.go

```diff
@@ -1643,6 +1643,28 @@ func (p *Predicate) Like(col, pattern string) *Predicate {
 	})
 }
 
+// Between is a shorthand for WHERE column BETWEEN ? AND ?
+func Between(col string, start, end interface{}) *Predicate {
+	p := P()
+	return p.Append(func(b *Builder) {
+		b.WriteString(fmt.Sprintf("%s BETWEEN ", col))
```

Comment:
Use `Ident` instead and add tests for both sqlite/mysql and postgres.

Reply:
done. like so:

```go
func Between(col string, start, end interface{}) *Predicate {
	p := P()
	return p.Append(func(b *Builder) {
		b.Ident(col)
		b.WriteString(" BETWEEN ")
		b.Arg(start)
		b.WriteString(" AND ")
		b.Arg(end)
	})
}
```

I also added test for sqlite on top of postgres and mysql.


Do you think i should add an `OpBetween` and `OpNotBetween` in `ops`? 

```go
	OpMul:     "*",
	OpDiv:     "/",
	OpMod:     "%",
 	OpBetween:     "BETWEEN",
	OpNotBetween:     "NOT BETWEEN",
```

So it can be this instead
```go
b.Ident(col)
b.WriteOp(OpBetween)
b.Arg(start)
```

Reply:
Also does this operation needs to be documented in say `crud.md`?